Makefile

```diff
@@ -1,5 +1,7 @@
 TAG ?= dev
 
+install-protoc:
+	sudo apt-get install -y protobuf-compiler
```

Comment:
This will not work in systems without sudo or apt-get.

Should we make it explicit in the CI definition instead of adding it to the makefile?

Reply:
This is only used in the github action. In the docker image this is done the same way.